src/plugins/ui/evaluation.ts

```diff
@@ -254,14 +255,26 @@ export class EvaluationPlugin extends UIPlugin {
         return result;
       }
 
+      let evalError: Error | undefined = undefined;
       // Performance issue: nested loop is faster than a map here
       for (let col = zone.left; col <= zone.right; col++) {
         const rowValues: ({ value: CellValue; format?: Format } | undefined)[] = [];
         for (let row = zone.top; row <= zone.bottom; row++) {
-          const cell = evalContext.getters.getCell(range.sheetId, col, row);
-          rowValues.push(cell ? getEvaluatedCell(cell) : undefined);
+          const cell = evalContext.getters.getCell(range.sheetId, col, row) as Cell | undefined;
+          let evalCell: { value: CellValue; format?: Format } | undefined = undefined;
+          try {
+            evalCell = cell ? getEvaluatedCell(cell) : undefined;
+          } catch (e) {
+            evalError = e;
```

Comment:
evalError will be the last error. Now it's the first error which is thrown, but not sure it's a big deal